### PR TITLE
Phase 0 updates.

### PR DESCRIPTION
**Pull request by devrimcavusoglu**
- rush config updated.
- MetricCallback updated/bugs fixed.
- Line-length set to 104 & code formatting.

### CODE REVIEW

**Comment by devrimcavusoglu** [comment]
@fcakyon Hocam değişiklikleri pushladım, müsait olduğun zaman bakarsın.

File: torchrush/metrics.py [hunk 1]
```diff
@@ -62,6 +64,9 @@ def __init__(
 
         self.labels = labels
         self.log_labelwise_metrics = log_labelwise_metrics
+        self._test_loss = []
```
Comment (fcakyon) [inline torchrush/metrics.py:67]: @devrimcavusoglu train lossu da eklesek daha iyi olmaz mi? train ile val setinin farkli dagilimlardan geldigi durumlarda en azindan train lossunun converge ettigini gorursek kodda bir hata olmadigini anlayabiliriz.

**Comment by fcakyon** [comment]
> @fcakyon Hocam değişiklikleri pushladım, müsait olduğun zaman bakarsın.

Bir yerle ilgili onerim var, onun disinda testler geciyorsa baska bir onerim yok. Eline saglik hocam :)

File: torchrush/metrics.py (hunk 1 above)
Comment (devrimcavusoglu) [reply torchrush/metrics.py:67]: Burdaki test ve val loss (attributeları) accumulate etmek için, train loss her batch'te loglanıyor şu anki şemada (böyle olacağını varsayıyorum), eğer bunun dışında bi kullanımımız olacaksa ekleyebiliriz.

Comment (fcakyon) [reply torchrush/metrics.py:67]: Bu haliyle uygun oyleyse 👍

**Review by fcakyon — APPROVED** [review]
